helpers.py

```diff
@@ -19,8 +19,10 @@ def mongo_to_dict(obj):
 
 
 def request_to_dict(request):
-    """Convert incoming flask requests into a dict"""
-    req_dict = request.values.to_dict()
+    """Convert incoming flask requests for objects into a dict"""
+    req_dict = request.values.to_dict(flat=True)
+    if request.is_json:
+        req_dict = request.get_json()  # get_dict returns python dictionary object
```

Comment:
I don't think I'll be mixing them. If it's RESTful to use GET requests, then we should do that.